packages/server/src/index.ts

```diff
@@ -12,6 +12,7 @@ import registerEndpoints from "./rest";
 import registerWebsockets from "./websockets";
 
 const writeFile = promisify(fs.writeFile);
+const exists = promisify(fs.exists);
```

Comment:
`fs.exists` is deprecated. From the docs:

> Stability: 0 - Deprecated: Use fs.stat() or fs.access() instead

You can reimplement `exists` as something like this:

```typescript
const stat = promisify(fs.stat);
const exists = async (filePath: string): Promise<boolean> => {
  try {
    await stat(filePath);
    return true;
  } catch (error) {
    if (error.code === "ENOENT") {
      return false;
    }
    throw error;
  }
}